core/templates/dev/head/domain/statistics/PlaythroughImprovementTaskObjectFactory.ts

```diff
@@ -60,6 +51,22 @@ angular.module('oppia').factory('PlaythroughImprovementTaskObjectFactory', [
         suggestions: PlaythroughIssuesService.renderIssueSuggestions(issue),
         playthroughIds: issue.playthroughIds,
       };
+
+      UserService.getUserInfoAsync().then(userInfo => {
+        if (!userInfo.isLoggedIn()) {
```

Comment:
So we want this to be visible to the user who doesn't have edit rights?

Reply:
Good point, updated to use permissions instead.

Done.